### ISSUE TITLE
Conjunctions with Non-Latin languages

### ISSUE THREAD

**Issue by PrimoQA** [michel-kraemer/citeproc-java#85]
Hello, thank you very much for the great API.
We are facing some issue on different languages.
When working with Non-Latin languages such as Russian or Hebrew, conjunctions such as '&' or 'et. al.' are not being translated and displayed in English.
for example APA CSL style:

Мария Владимировна Певная, **&** Анна Александровна Кузьминчук (2014). ИСТОРИЯ АМЕРИКАНСКОГО ВОЛОНТЕРСТВА В СОЦИОЛОГИЧЕСКОМ РАКУРСЕ Sovremennye Issledovaniâ Socialʹnyh.

We've tried different language formats ( "heb", "he" or "he-IL" for hebrew) but still getting the same response.

Below is an example for parameters on the API request.
The CSLItemData object sent to the API for the language format "heb":

{"id":"-GEN-1as39vqkk5","type":"book","language":"heb","author":[\{"family":"כתרי, רון","given":""}
,{"family":"מלצר, אפי","given":""}],"issued":{"date-parts":[[2008]]},"container-title":"מבט מל"מ : כתב עת לענייני מודיעין ובטחון מבית המרכז למורשת המודיעין","DOI":"","edition":"","ISBN":"","ISSN":"","issue":"52","keyword":"מבט מלמ (המרכז למורשת המודיעין)","note":"","number-of-pages":"","page":"","page-first":"","PMID":"","publisher":"המרכז למורשת המודיעין (מ.ל.מ) ע"ר","publisher-place":"","section":"","title":"גיליון 52","URL":"","volume":""}

Can you please assist in solving this?
Thanks in advance.

**Comment by nilinatan** [comment]
Hi Michael,

Hope you are doing ok.
will you help us with the issue stated above. we are using the API and encounter issues with Non-Latin languages not translating terms such as  'et. al.'

If you please check the example above and let us know if this a known issue that can be fixed.

Best Regards 
Nili Natan
Product Manager of Discovery & Delivery 
nili.natan@exlibrisgroup.com
www.exlibrisgroup.com

**Comment by michel-kraemer** [comment]
@PrimoQA @nilinatan Please note that the APA style defines that the ampersand symbol should be used to separate multiple names: https://github.com/citation-style-language/styles/blob/master/apa.csl#L331
As far as I know, this symbol cannot be overridden in a locale file (and the Hebrew file `he-IL.xml` in fact does not do so).

Also note that the APA style requires at least 21 names before it displays "et al.": https://github.com/citation-style-language/styles/blob/master/apa.csl#L1878

If you keep these two points in mind everything should work as expected. Please let me know if this helps. Thanks.

**Comment by PrimoQA** [comment]
Thanks for your reply, we will process that.

**Comment by PrimoQA** [comment]
Hello Michel, thank you for your help.
We've tried a few test based on your answer, we've tried using more than 21 names in APA style or other styles or other number of names, and in all cases the conjunctions were not translated, "&" or "et al" was displayed.
In order to set the language we've the CSLItemDataBuilder.language(String language), with different language formats ( "heb", "he" or "he-IL" ) but still got the same response.
Is there any action we are missing? we see that the term exists in the [locale file](https://github.com/citation-style-language/locales/blob/master/locales-he-IL.xml) but we still can't get the translation: 
    <term name="et-al">ואחרים</term>

Thanks,
Primo team
